com.unity.sg2/Editor/GraphDeltaRegistry/PreviewManager/IPreviewUpdateReceiver.cs

```diff
@@ -0,0 +1,9 @@
+﻿using UnityEngine;
+
+namespace UnityEditor.ShaderGraph.GraphDelta
+{
+    public interface IPreviewUpdateReceiver
```

Comment:
Rename to `IPreviewDataStore` or something less conflicting with 'IPreviewUpdateListener'